src/usb-backend.js

```diff
@@ -60,6 +61,80 @@ function getMatchingDeviceFilters(device, filters) {
     }).filter(filterName => filterName);
 }
 
+
+/**
+ * Given a libusb error, this function assigns the error argument an error code
+ * representing the error type.
+ *
+ * @param {Object} err The error to assign an error code to.
+ * @returns {Object} The error with a code assigned, given it is a libusb error.
+*/
+function decorateError(err) {
+    const error = err;
+    switch (error.message) {
+        case 'LIBUSB_SUCCESS': {
+            error.errorCode = ErrorCodes.LIBUSB_SUCCESS;
+            break;
+        }
```

Comment:
I disagree. There is a direct mapping for now, but there is no guarantee this will be the case in the future. The point of using error codes is to not rely on the error messages directly. If for some reason libusb starts throwing error messages in German, we would have to add checks for that as well.

Reply:
Good point :+1: